packages/cascara/src/modules/ActionButton/ActionButton.js

```diff
@@ -5,12 +5,14 @@ import { ModuleContext } from '../context';
 import { Button } from 'reakit';
 
 const propTypes = {
-  /** Every action can have a name */
+  /** DEPRECATED - do not use */
   actionName: pt.string,
   /** Presents the button without a label. NOT USER CONFIGURABLE */
```

Comment:
@lordpixel I wonder if we should consider props like this and possibly label them as private. We could then use that information to omit them from the docs. Additionally, we can also then write some functions to allow us to determine which props we want to handle and which are outside of that scope. It will be easier to use functions to pass those handled props where they need to go and let others pass through. This we can also use to protect private props.

Reply:
```suggestion
  /** PRIVATE: Shows a label */
```

Reply:
I like the approach you propose @brianespinosa, let's explore how we can tackle this.

The reasonn for this particular prop to be marked as `deprecated` is because we renamed it to just `name`. We still need the old `actionName` at least until deprecation is fully applied.